### PR TITLE
Adds chevron to folder in column view

### PR DESCRIPTION
**Pull request by cinqmilleans**
Adds chevron to folder in column view. #9546

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
FolderChevron_0
FolderChevron_1

### CODE REVIEW

**Comment by mdtauk** [comment]
Might you consider changing the chevron to use the secondary or tertiary brushes.  They don't need to be quite so prominent because they are not directly interactable compared to the other items in the list.

**Comment by cinqmilleans** [comment]
I don't know what the secondary or tertiary brush is. Here is a suggestion with opacity 0.6.
FolderChevron_2
FolderChevron_3

File: src/Files.Uwp/Views/LayoutModes/ColumnViewBase.xaml [hunk 1]
```diff
@@ -332,6 +333,13 @@
                                                 Visibility="{x:Bind FileTagsUI, Converter={StaticResource EmptyObjectToObjectConverter}, Mode=OneWay}" />
                                         </StackPanel>
                                     </Grid>
+                                    <FontIcon
+                                        Grid.Column="3"
+                                        HorizontalAlignment="Right"
+                                        VerticalAlignment="Center"
+                                        FontSize="12"
+                                        Glyph="&#xE76C;"
+                                        Visibility="{x:Bind IsFolder}" />
```
Comment (yaichenbaum) [inline src/Files.Uwp/Views/LayoutModes/ColumnViewBase.xaml:342]:
```suggestion
                                        x:Load="{x:Bind IsFolder}" />
```

**Comment by yaichenbaum** [comment]
Note to self to test this with Accessibility Insights

**Comment by cinqmilleans** [comment]
- The resources I'm being asked to use don't support the dark theme. It is not possible to have an adequate color in the 2 themes.
- Synchronization and color tag information is important and is available in other views. Perhaps it should be displayed before or after the text.

**Comment by yaichenbaum** [comment]
> The resources I'm being asked to use don't support the dark theme. It is not possible to have an adequate color in the 2 themes.

I'll investigate this and send you the proper resource to use

> Synchronization and color tag information is important and is available in other views. Perhaps it should be displayed before or after the text.

This information can be displayed in a tooltip and the details pane but it doesn't fit in the file area anymore as it would be too crowded with it.

**Comment by yaichenbaum** [comment]
@cinqmilleans does `TextFillColorSecondary` work?

**Comment by yaichenbaum** [comment]
@cinqmilleans can you push the code so I can point you in the right direction? I think you might be using `StaticResource` instead of `ThemeResource`.

**Comment by cinqmilleans** [comment]
Your answer helped me. I was using StaticResource instead of ThemeResource. With ThemeResource, it's ok in both themes. I changed the pr to use TextFillColorTertiary, because TextFillColorSecondary is not discrete enough.

**Comment by yaichenbaum** [comment]
Can post an updated screenshot?

**Comment by cinqmilleans** [comment]
Here.
Chevron1
Chevron2
Chevron3
Chevron4

**Review by yaichenbaum — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@cinqmilleans oh, one more thing before merging, did you test that the chevrons don't get in the way of the screen reader?

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**